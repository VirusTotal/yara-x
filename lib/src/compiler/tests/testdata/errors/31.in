rule test {
  condition:
    for 1 n in (1, 2, "3") : (
      n == 2
    )
}