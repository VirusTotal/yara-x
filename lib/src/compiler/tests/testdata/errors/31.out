error: mismatching types
 --> line:3:20
  |
3 |     for 1 n in (1, 2, "3") : (
  |                    ^ this expression is `integer`
  |                       ^^^ this expression is `string`
  |